packages/.coderabbit.yaml

```
@@ -0,0 +1,196 @@
+    suggested_labels: false # Don't suggest labels
+    auto_apply_labels: false # Don't auto-apply suggested labels
+    suggested_reviewers: false # Don't suggest reviewers
+    poem: false # Don't generate poems in walkthrough
+    labeling_instructions: [] # No custom labeling instructions
+    path_filters: # Glob patterns for files to include/exclude
+        - "!dist/**, !node_modules/**" # Exclude dist and node_modules directories
+    path_instructions:
+        - path: "**/*.ts"
+          instructions: |2-
+          Review TypeScript files for adherence to the following guidelines:
+                - Avoid over-abstraction; prioritize composition over inheritance.
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 14 but found 16

(indentation)